packages/common-web/src/arrays.ts

```diff
@@ -21,3 +21,7 @@ export const mapDefined = <T, S>(
   }
   return output
 }
+
+export const filterDefined = <T>(arr: (T | undefined)[]): T[] => {
+  return arr.filter((item) => item !== undefined) as T[]
+}
```

Comment:
I think we need to take a little care now to ensure we include changesets for changes to deps, so that the PDS distro will rely on a proper published version of its deps.

Reply:
Ended up just scrapping this util func and using a type annotation instead 👌 

(most recent TS version actually fixes the need for this 👀)